projects/kit/components/avatar/avatar.component.ts

```diff
@@ -35,6 +35,10 @@ export class TuiAvatarComponent {
     @tuiDefaultProp()
     text = '';
 
+    @Input()
+    @tuiDefaultProp()
+    fallback: SafeHtml | string | null = null;
```

Comment:
Add new input's description to https://taiga-ui.dev/components/avatar/API

Reply:
fixed